installer/datafiles/base_omsagent.data

```diff
@@ -95,16 +95,12 @@ MAINTAINER:              'Microsoft Corporation'
 /opt/microsoft/omsagent/plugin/security_lib.rb;                         source/code/plugins/security_lib.rb;                   744; root; root
```

Comment:
why is this commit part of the PR?

Reply:
I think something wrong with the diff but the changes look right